package.json

```
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "devDependencies": {
+    "@wordpress/scripts": "^28.4.0"
+  },
+  "scripts": {
+    "lint-docs": "npx wp-scripts lint-md-docs ."
```

Please use the [company standard](https://inpsyde.atlassian.net/wiki/spaces/CN/pages/4856086552/WP+Scripts+configs) of naming the script.
```suggestion
    "lint:md": "wp-scripts lint-md-docs ."
```
Please ignore the `--ignore` flags in the documentation; these are outdated. Also, similar to above, is the `.` really needed?

---

Changed with https://github.com/inpsyde/reusable-workflows/pull/149/commits/05ef594c1c38db04c1690816335ea06e00ce2431.

`.` was indeed redundant.